### PR TITLE
import_page plugin

### PR DESCRIPTION
**Pull request by ralsina**
Check the README. Still needs work to have  a clean/useful interface.

### CODE REVIEW

File: v7/import_page/import_page.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © 2014, 2015 Miguel Ángel García
```
Comment (Kwpolska) [inline v7/import_page/import_page.py]: ?

File: v7/import_page/import_page.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    doc_usage = "[options] page_url [page_url,...]"
+    doc_purpose = "import arbitrary web pages"
+
+    def _execute(self, options, args):
+        """Import a Page."""
+        for url in args:
+            self._import_page(url)
+
+    def _import_page(self, url):
+        r = requests.get('http://en.wikipedia.org/wiki/Information_extraction')
```
Comment (Kwpolska) [inline v7/import_page/import_page.py]: ?

File: v7/import_page/import_page.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+.. title: {title}
+.. slug: {slug}
+-->
+
+{content}
+'''
+
+
+class CommandImportPage(Command):
+    """Import a Page or Octopress blog."""
```
Comment (Kwpolska) [inline v7/import_page/import_page.py]: ?

File: v7/import_page/import_page.py (hunk 1 above)
Comment (ralsina) [inline v7/import_page/import_page.py]: Copypasta.

**Pull request merged**